app/Service/Student/UpdateStudentProfileService.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Service\Student;
+
+use App\Models\Student;
+use App\Exceptions\{
+    StudentNotFoundException,
+    ResumeNotFoundException
+};
+
+class UpdateStudentProfileService
+{
+    public function execute(string $studentId, array $data): bool
+    {
+        return $this->updateStudentProfile($data, $studentId);
+    }
+
+    public function updateStudentProfile(array $data, string $studentId):bool
```

Comment:
change the order of the parameters (first the subject, then the action)

Reply:
done